app/Main.hs

```diff
@@ -1,6 +1,6 @@
 module Main where
 
-import           Init (runApp)
+import Init (runApp)
```

Comment:
We no longer indent so that module names are aligned. This helps keep the column count low, and makes it easier to just type this out manually without worrying about alignment.